src/components/Disputes/actions/DisputeVoting.js

```diff
@@ -102,8 +114,56 @@ function VotingActions({ canJurorVote, onRequestCommit }) {
   )
 }
 
-const VotingButton = styled(Button)`
-  width: calc((100% - ${2 * GU}px) / 3);
+const RefuseToVoteHint = ({ compactMode, width }) => {
+  const theme = useTheme()
+
+  const Container = compactMode ? Label : 'div'
+
+  return (
+    <div
+      css={`
+        display: flex;
+        align-items: center;
+        justify-content: ${compactMode ? 'center' : 'flex-end'};
+        margin-bottom: ${1.5 * GU}px;
+      `}
+    >
+      <div
+        css={`
+          width: ${width};
```

Comment:
Do we need to set the width of this component, if we've already applied `flex-end`?

Reply:
Yes we do, because we want the hint to be centered and aligned relative to the refuse to vote hint on big scale displays